### ISSUE TITLE
Unsplash source not working (404)

### ISSUE THREAD

**Issue by fockr** [Simonmicro/better-backgrounds#26]
Unsplash as image source stopped working some time ago with changing error codes - currently with "`Could not download image (404)`" when trying to download a new image.

What could be the reason?

**Comment by mmck345** [comment]
![image](https://github.com/user-attachments/assets/2dae5787-eaa4-4b69-9d9e-fff909eba5e5)
Are you sure you set the source for images in better background settings?

**Comment by simonmicro** [comment]
Yeah, Unsplash are not nice and broke their deprecated `source.unsplash.com` endpoint. The new one explicitly disallows the use of e.g. background image changer tools, so I was not motivated enough to integrate with their API (using the users provided API-key). I did some experiments and it worked, but never cleaned it up and published that.
